backend/tests/data_layer/database/crud/sqlite/test_smartapi_crud.py

```
@@ -48,21 +48,38 @@
-        result = get_smartapi_tokens_by_all_conditions(symbol="INFY", exchange="NSE")
-        assert token1.to_dict() == result[1].to_dict()
+        # Test get_smartapi_tokens_by_all_conditions with valid conditions
+        result = get_smartapi_tokens_by_all_conditions(
+            symbol="INFY", exchange="NSE", instrument_type="EQ"
+        )
+        assert token1.to_dict() == result[0].to_dict()
 
+        # Test get_smartapi_tokens_by_all_conditions with different symbol and exchange
         result = get_smartapi_tokens_by_all_conditions(symbol="SBI", exchange="BSE")
         for item in result:
             assert "SBI" == item.symbol
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `item.symbol == "SBI"`

(SIM300)